lireincore/imgcache-bundle/0.1/config/packages/lireincore_imgcache.yaml

```diff
@@ -0,0 +1,14 @@
+lireincore_imgcache:
+    driver: gd
+#    srcdir: '%kernel.project_dir%/path/to/source/images'
+    destdir: '%kernel.project_dir%/public/thumbs'
+    webdir: '%kernel.project_dir%/public'
+#    baseurl: 'https:\\sitename.com'
```

Comment:
you should remove the comments here, if you would like to provide additional information you can add a text file which will be prompted after the installation (and btw. it should be `https://`, not `\\`

Reply:
I agree with @OskarStark Please remove the config in comments.

Reply:
done